protoc-gen-swagger/options/annotations_v3.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package grpc.gateway.protoc_gen_openapi.options;
+
+option go_package = "github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger/options";
+
+import "protoc-gen-swagger/options/openapiv3.proto";
+import "google/protobuf/descriptor.proto";
+
+extend google.protobuf.FileOptions {
+  // ID assigned by protobuf-global-extension-registry@google.com for grpc-gateway project.
+  //
+  // All IDs are the same, as assigned. It is okay that they are the same, as they extend
+  // different descriptor messages.
+  OpenAPI openapiv3 = 1042;
```

Comment:
Despite what is may seem the comment says, it's actually not encouraging keeping the same ID for two completely different messages. This is particularly true for v2 to v3 transition; it's not only incompatible, but there's also no reason why two generators would not coexist.